homeassistant/components/lifx/util.py

```diff
@@ -45,6 +45,17 @@ def async_get_legacy_entry(hass: HomeAssistant) -> ConfigEntry | None:
     return None
 
 
+def infrared_brightness_value_to_option(value: int) -> str | None:
+    """Convert infrared brightness from value to option."""
+    return INFRARED_BRIGHTNESS_VALUES_MAP.get(value, None)
```

Comment:
`None` is the default value returned if the key is missing in the dict.

Reply:
Right.